lib/Doctrine/DBAL/Connection.php

```diff
@@ -1256,12 +1254,12 @@ public function beginTransaction()
                 $logger->stopQuery();
             }
         }
+
+        return true;
     }
 
     /**
-     * Commits the current transaction.
-     *
-     * @return void
+     * {@inheritDoc}
```

Comment:
We should get rid of this in future - inheritance is implicit.

Reply:
What do you mean? It explicitly implements the driver connection interface and therefore inherits its API. Are you talking about not implementing the driver connection interface in the wrapper connection in the future?

Reply:
I mean that `{@inheritDoc}` is redundant - inheritance is implicit and this shouldn't be needed. But we need to first address this on CS level, I also discussed this with @kukulich somewhere.

Reply:
Yeah, this is true. It shouldn't be needed according to [phpDocumentor](http://docs.phpdoc.org/guides/inheritance.html#methods). Looks like PhpStorm doesn't agree with that ([WI-38168](https://youtrack.jetbrains.com/issue/WI-38168)).